mirage/serializers/preprint-provider.ts

```diff
@@ -0,0 +1,73 @@
+import { ModelInstance } from 'ember-cli-mirage';
+import config from 'ember-get-config';
+import PreprintProvider from 'ember-osf-web/models/preprint-provider';
+import ApplicationSerializer, { SerializedRelationships } from './application';
+
+const { OSF: { apiUrl } } = config;
+
+export default class PreprintProviderSerializer extends ApplicationSerializer<PreprintProvider> {
+    buildNormalLinks(model: ModelInstance) {
+        return {
+            self: `${apiUrl}/v2/providers/preprints/${model.id}/`,
+        };
+    }
+
+    buildRelationships(model: ModelInstance<PreprintProvider>) {
+        const relationships: SerializedRelationships<PreprintProvider> = {
+            subjects: {
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/providers/preprints/${model.id}/subjects/`,
+                        meta: this.buildRelatedLinkMeta(model, 'subjects'),
+                    },
+                },
+            },
+            highlightedSubjects: {
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/providers/preprints/${model.id}/highlightedSubjects/`,
+                        meta: this.buildRelatedLinkMeta(model, 'highlightedSubjects'),
+                    },
+                },
+            },
+            licensesAcceptable: {
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/providers/preprints/${model.id}/licenses/`,
+                        meta: {},
+                    },
+                },
+            },
+            moderators: {
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/providers/preprints/${model.id}/moderators/`,
+                        meta: this.buildRelatedLinkMeta(model, 'moderators'),
+                    },
+                },
+            },
+            preprints: {
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/providers/preprints/${model.id}/preprints/`,
+                        meta: {},
+                    },
+                },
+            },
+            // TODO: subscriptions when we move ember-osf-reviews¥
+        };
+
+        if (model.brand) {
+            relationships.brand = {
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/brands/${model.brand.id}/`,
+                        meta: {},
+                    },
+                },
+            };
+        }
```

Comment:
So, to be sure, a `brand`  relationship is being added to the backend, and will have the CSS as well? Are we going to be maintaining the CSS two ways, one in the admin app, and one via the preprint provider GitHub project?

Reply:
I think that's the idea now until all of preprint is in `ember-osf-web`.

Reply:
Yea, currently we are doing both ways of branding: using the `brand` relationship, and the custom css file. There's some stuff that the `brand` relationship can't currently do (navbar customization), so I've opted to use both and try to make sure the custom css route isn't utilized further into the future.